source/frontend/src/features/leases/add/AddLeaseContainer.tsx

```diff
@@ -36,21 +38,28 @@ export const AddLeaseContainer: React.FunctionComponent<
   const { search } = useMapSearch();
 
   useEffect(() => {
+    if (!!selectedFileFeature && !!formikRef.current) {
+      formikRef.current.resetForm();
+      formikRef.current?.setFieldValue('properties', [
+        PropertyForm.fromMapProperty(mapFeatureToProperty(selectedFileFeature)),
+      ]);
+    }
     return () => {
```

Comment:
Hmm, I think this should still only fire once, but am not sure. Actually I guess it isn't a big deal if this fires more then once as long as the create file from map functionality still works. Carry on.